ivy/functional/frontends/tensorflow/linalg.py

```diff
@@ -470,3 +470,8 @@ def tensor_diag_part(input, name=None):
     reshaped = ivy.reshape(input, (prod, prod))
     diagonal = ivy.diagonal(reshaped)
     return ivy.reshape(diagonal, tuple(half_shape))
+
+
+@to_ivy_arrays_and_back
+def lu_solve(lower_upper, perm, rhs, validate_args=False, name=None):
+    return ivy.solve(lower_upper, perm, rhs, validate_args=False, name=None)
```

Comment:
`ivy.solve()` takes on 2 arguments, also I'd suggest you should read the `docstring` on how does this function work.
I'd suggest you implement this compositionally for now. If you don't know what compositionally means I'd refer you to our deep dive in the docs.

Reply:
I can't find any information like this in the deep dive doc, please. Can you help?

Reply:
@zaeemansari70 Please check what I just did!

Reply:
Here you go, this might help you :slightly_smiling_face: https://unify.ai/docs/ivy/docs/functional/ivy/linear_algebra/ivy.functional.ivy.linear_algebra.solve.html#solve